index.html

```diff
@@ -229,10 +229,10 @@ <h3 class="hero-text hero-text-white">
                 If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
             </h3>
 
-            <form action="" class="contact-form">
-                <input type="text" name="" id="" placeholder="Yeremias" >
-                <input type="email" name="" id="" placeholder="natajayanj14@gmail.com" >
-                <textarea name="" id="" cols="30" rows="10" placeholder="Write your message here" ></textarea>
+            <form action="https://formspree.io/f/xjvdwrqb" method="POST" class="contact-form">
+                <input required type="text" name=""  id="" placeholder="Ingresa tu nombre" maxlength="30" >
+                <input required type="email" name="" id="" placeholder="Ingresa tu correo" >
+                <textarea name="coment" cols="30" rows="10" placeholder="Ingresa tus comentarios" maxlength="500"></textarea>
```

Comment:
Please, kindly ensure that the `textarea` is also `required` as specified in the requirements 😉.